### PR TITLE
1588 accidental unpacking during evaluating the if statement

### PR DESCRIPTION
**Pull request by FYL2003**
Before this change, the if statement inside a block statement is not value producing, details in the following video
https://github.com/source-academy/js-slang/assets/97390773/d1300c04-df1f-4f9c-8a50-9b0dcabe3d69

After the change, the if statement inside a block statement is value producing and will not accidentally unpack itself.

https://github.com/source-academy/js-slang/assets/97390773/c39527cb-06f1-4e5a-ae9e-bce52a26577e

### CODE REVIEW

File: src/createContext.ts [hunk 1]
```diff
@@ -11,9 +11,9 @@ import { list_to_vector } from './stdlib/list'
 import { listPrelude } from './stdlib/list.prelude'
 import { localImportPrelude } from './stdlib/localImport.prelude'
 import * as misc from './stdlib/misc'
-import * as pylib from './stdlib/pylib'
 import { nonDetPrelude } from './stdlib/non-det.prelude'
 import * as parser from './stdlib/parser'
+import * as pylib from './stdlib/pylib'
```
Comment (martin-henz) [inline src/createContext.ts:16]: Is this a mistake? I don't see why you would add pylib.

This might be the reason why CI fails.

**Review by martin-henz — CHANGES_REQUESTED** [review]

File: src/createContext.ts (hunk 1 above)
Comment (FYL2003) [reply src/createContext.ts:16]: I did not do anything with this file, maybe my last git pull add it accidentally. So I need to remove this line and submit again?

Comment (martin-henz) [reply src/createContext.ts:16]: Yes, please remove the line from your PR.

Comment (RichDom2185) [reply src/createContext.ts:16]: It's not a mistake, pylib wasn't "added", the import line was just rearranged (see the deleted line above the added line) - probably due to linter. Just rearrange it back, please. Thanks!

**Review by martin-henz — APPROVED** [review]
All good now.

**Pull request merged**